apis/duck/typed_test.go

```diff
@@ -73,7 +74,7 @@ func TestSimpleList(t *testing.T) {
 
 	// This hangs without:
 	// https://github.com/kubernetes/kubernetes/pull/68552
-	_, lister, err := tif.Get(SchemeGroupVersion.WithResource("resources"))
+	_, lister, err := tif.Get(context.Background(), SchemeGroupVersion.WithResource("resources"))
```

Comment:
This might be more correct by using a `context.WithCancel` instead of the stopCh above. You can then use `ctx.Done()` as the StopChannel and pass the respective context here.

Reply:
done in 77678bb83f220dd422bf886e7ce009f6f23d6616 
PTAL if you like that solution